web/eslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```

Comment:
its been like it for at least a month
https://github.com/incuna/angular-feincms/blob/master/.eslintrc